common/notify_stream/src/lib.rs

```diff
@@ -90,27 +92,29 @@ pub enum RecursiveMode {
 pub struct Watcher {
     watcher: OsWatcher,
     rx: Rc<async_channel::Receiver<Result<notify::Event, notify::Error>>>,
+    tx: Rc<async_channel::Sender<Result<notify::Event, notify::Error>>>,
```

Comment:
Was the purpose for adding Sender to the Watcher struct for testing purposes?

Reply:
for testing.